.github/workflows/collector.yml

```diff
@@ -171,7 +171,13 @@ jobs:
           username: ${{ secrets.QUAY_STACKROX_IO_RW_USERNAME }}
           password: ${{ secrets.QUAY_STACKROX_IO_RW_PASSWORD }}
 
-      - name: Create and push multiarch manifest for stackrox-io -slim
+      - name: Create and push multiarch manifest for stackrox-io
+        uses: ./.github/actions/create-multiarch-manifest
+        with:
+          base-image: quay.io/stackrox-io/collector:${{ inputs.collector-tag }}
```

Comment:
Do we also need to add the `-latest` variants for both stackrox-io and rhacs-eng here?